docs/consume-packages/install-use-packages-nuget-cli.md

```diff
@@ -1,118 +1,123 @@
+Use the `-OutputDirectory` option to install packages to a specific directory. If you don't specify an output directory, `install` uses the current directory.
 
-2. Use the following command to install a NuGet package to the *packages* folder.
+```cli
+nuget install <packageID | configFilePath> -OutputDirectory <outputDirectory>
+```
 
-    ```cli
-    nuget install <packageID> -OutputDirectory packages
-    ```
+For example, to install the `Newtonsoft.json` package to the *packages* subdirectory, use the following command:
 
-    To install the `Newtonsoft.json` package to the *packages* folder, use the following command:
+```cli
+nuget install Newtonsoft.Json -OutputDirectory packages
+```
+
+Instead of specifying a package to install, you can specify an existing *packages.config* file in the current or another directory. The `install` command installs all the packages listed in the *packages.config* file.
 
-    ```cli
-    nuget install Newtonsoft.Json -OutputDirectory packages
-    ```
+```cli
+nuget install packages.config
+```
 
-Alternatively, you can use the following command to install a NuGet package using an existing `packages.config` file to the *packages* folder. This does not add the package to your project dependencies, but installs it locally.
+For example, the following command installs all the packages listed in *packages.config* in the *config* subdirectory to the *packages* subdirectory:
 
 ```cli
-nuget install packages.config -OutputDirectory packages
+nuget install config\packages.config -OutputDirectory packages
+
 ```
 
 ## Install a specific version of a package
 
-If the version is not specified when you use the [install](../reference/cli-reference/cli-ref-install.md) command, NuGet installs the latest version of the package. You can also install a specific version of a Nuget package:
+The `install` command installs the latest version of a package unless you specify a different version. To install a specific version of a package, use the `-Version` option:
 
 ```cli
 nuget install <packageID | configFilePath> -Version <version>
 ```
 
-For example, to add version 12.0.1 of the `Newtonsoft.json` package, use this command:
+For example, to install version 12.0.1 of the `Newtonsoft.json` package, use:
 
 ```cli
 nuget install Newtonsoft.Json -Version 12.0.1
 ```
 
-For more information on the limitations and behavior of `install`, see [Install a package](#install-a-package).
-
-## Remove a package
-
-To delete one or more packages, delete the packages you want to remove from the *packages* folder.
-
-If you want to reinstall packages, use the `restore` or `install` command.
-
 ## List packages
 
-You can display a list of packages from a given source using the [list](../reference/cli-reference/cli-ref-list.md) command. Use the `-Source` option to restrict the search.
+Use the [list](../reference/cli-reference/cli-ref-list.md) command to display a list of packages. Use the `-Source` option to restrict the list.
```

Comment:
I  think we should tweak this a bit to indicate that this shows the packages installed in the packages folder.